bot/exts/holidays/halloween/candy_collection.py

```diff
@@ -173,6 +173,7 @@ async def send_no_candy_spook_message(
     async def candy(self, ctx: commands.Context) -> None:
         """Get the candy leaderboard and save to JSON."""
         records = await self.candy_records.items()
+        user = await self.bot.fetch_user(ctx.author.id)
```

Comment:
We don't need to fetch the user here, we are using the user object to get the id/mention which can be done on `ctx.author` also, the two are the same objects.